apps/cli/src/commands/staxRemoveImage.ts

```diff
@@ -6,6 +6,6 @@ import { withDevice } from "@ledgerhq/live-common/hw/deviceAccess";
 
 export default {
   description: "Remove custom lock screen",
-  arg: [deviceOpt],
+  args: [deviceOpt],
```

Comment:
this modification fixs the cli command.
otherwise, there is an error when we launch the "pnpm run:cli" command due to "cmd.args" variable is undefined in the "cil/src/cli.ts" file